code/game/objects/items/devices/atomic_disassembler.dm

```diff
@@ -0,0 +1,156 @@
+/obj/item/device/atomic_disassembler
+	name = "P.A.D.D."
+	desc = "The Portable Atomic Disassembly Device (patent pending) is a revolutionary device in breaking down objects at the atomic level. To use, you must attach a condensed matter canister and a power \
```

Comment:
Holy description batman.

I challenge you to find another item in game with a description as long as this.
If you win the challenge you can keep this ridiculous description.
If you loose/choose not to play, you should make it short+sweet, like every other description.